aiounifi/controller.py

```diff
@@ -190,5 +199,10 @@ def message_handler(self, message: dict) -> dict:
 
 def _raise_on_error(data):
     """Check response for error message."""
-    if isinstance(data, dict) and data["meta"]["rc"] == "error":
-        raise_error(data["meta"]["msg"])
+    if isinstance(data, dict):
```

Comment:
Again, maybe there's a difference in our setups, but mine raised exception without null-safe property access. Also noticed not every error response came in the format `"data.meta.rc": "error"`, so this catches the other format I experienced (_i.e._ invalid password)

Reply:
Those other errors are probably UniFi os specific errors. How did you get them?

Reply:
Most of them were on login. Using bad credentials, trying to log into an account with 2FA enabled--things like that. Didn't keep track of which ones specifically, but I do remember logging in with bad credentials returned a response with an `Errors` array and no `meta` object.

Reply:
So many cases to cover. Long term I should probably rewrite parts of it...

Reply:
So unfortunate that they don't provide documentation on their API...

Reply:
You don't happen to have a raw example of this?

Reply:
I don't.. Going back through it though, it looks like a commit you had pushed just before I opened this PR had already fixed that--at least with invalid logins. If I rebase off the commit after this one, I don't have that issue any longer.

Reply:
Well great then. Thanks for your help! I've reflected your changes in the already merged PR, so I'll close this one.